swagger.json

```diff
@@ -4581,6 +4587,15 @@
                     }
                 }
         },
+        "TagIds": {
+            "type": "array",
+            "description": "A list of tag IDs.",
```

Comment:
I think we should have done this for contactIds too, but can we describe how this list works on an update? Does the list itself behave like a `PUT` where I need to supply the complete list?

Reply:
It does, but that's more a function of the endpoint than the actual object. I can add a note about that to the address PATCH description.